ui/raidboss/data/05-shb/alliance/the_tower_at_paradigms_breach.js

```diff
@@ -0,0 +1,490 @@
+      netRegex: NetRegexes.headMarker({ id: '00DA' }),
+      run: function(data, matches) {
+        data.busterTargets = data.busterTargets || [];
+        data.busterTargets.push(matches.target);
+      },
+    },
+    {
+      id: 'Paradigm Knave Magic Artillery Beta Not You',
+      netRegex: NetRegexes.headMarker({ id: '00DA', capture: false }),
+      delaySeconds: 0.5,
+      durationSeconds: 4,
+      suppressSeconds: 5,
+      infoText: function(data, _, output) {
+        if (!data.busterTargets)
+          return;
+        if (data.busterTargets.includes(data.me))
+          return;
+
+        if (data.role === 'healer')
+          return output.tankBuster();
+
+        return output.avoidTankBuster();
+      },
+      run: (data) => delete data.busterTargets,
+      outputStrings: {
+        tankBuster: {
+          en: 'Tank Buster',
+          de: 'Tank buster',
+          fr: 'Tank buster',
+          ja: 'タンクバスター',
+          cn: '坦克死刑',
+          ko: '탱버',
+        },
+        avoidTankBuster: {
+          en: 'Avoid tank buster',
+          de: 'Tank buster ausweichen',
+          fr: 'Évitez le tank buster',
+          ja: 'タンクバスターを避ける',
+          cn: '远离坦克死刑',
+          ko: '탱버 피하기',
+        },
+      },
+    },
+    {
+      id: 'Paradigm Knave Magic Artillery Alpha Spread',
+      netRegex: NetRegexes.headMarker({ id: '00A9' }),
+      condition: Conditions.targetIsYou(),
+      durationSeconds: 4,
+      response: Responses.spread(),
+    },
+    {
+      id: 'Paradigm Knave Lunge',
+      netRegex: NetRegexes.startsUsing({ id: '5EB1', source: 'Knave of Hearts', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5EB1', source: 'Herzbube', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5EB1', source: 'Jack', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5EB1', source: 'ジャック', capture: false }),
+      durationSeconds: 5,
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Knockback (from boss)',
+          de: 'Rückstoß (vom Boss)',
+        },
+      },
+    },
+    {
+      id: 'Paradigm Copied Knave Lunge',
+      netRegex: NetRegexes.startsUsing({ id: '5EB1', source: 'Copied Knave', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5EB1', source: 'Kopiert(?:e|er|es|en) Herzbube', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5EB1', source: 'Réplique De Jack', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5EB1', source: '複製サレタジャック', capture: false }),
+      condition: (data) => !data.cloneLunge,
+      durationSeconds: 5,
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Knockback (from clone)',
+          de: 'Rückstoß (vom Klon)',
+        },
+      },
+      run: (data) => data.cloneLunge = true,
+    },
+    {
+      id: 'Paradigm Copied Knave Lunge Out of Middle',
```

Comment:
Is there a Lunge Get Middle?

Reply:
Don't know! There's `Assuming one Lunge + Colossal Impact pattern; needs confirmation` in the TODO